pkg/actuators/machine/machine_scope_test.go

```diff
@@ -241,27 +214,24 @@ func TestPatchMachine(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			timeout := 10 * time.Second
-			gs := NewWithT(t)
```

Comment:
Should keep this sub gomega, so that you fail the individual test case rather than the whole test if there's a failure